### PR TITLE
feat(TFD-477): Add RowGenerator component.

### PR DESCRIPTION
**Pull request by RyanSkraba**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [X] Tests for the changes have been added (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

Adds a RowGenerator component for Beam execution environment.

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/884/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/884/)

File: components/components-localio/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* Number of partitions
+* Number of rows per partition
+
+### Numeric annotations
+
+The following annotations can be added to the schema for `INT`, `LONG`, `FLOAT` and `DOUBLE` types:
+
+| name | value | 
+|------|-------|
+| guassian |          |
```
Comment (RyanSkraba) [inline components/components-localio/README.md]: These are TODO -- I wanted to have the documentation for the component at the root of the component module, but most of this work is in the beam-related part in components-adapter-beam.  I'll update the doc here with a TODO tag and we can decide what to do with internal documentation.

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/909/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/909/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/913/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/913/)

**Pull request merged**